Peer.Domain/Models/PeerPullRequest.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Peer.Domain.Models
+{
+    public class PeerPullRequest
+    {
+        private string _title;
+        private object _assignee;
+
+        public PeerPullRequest(string Title, object Assignee)
```

Comment:
Already in the peer namespace, any reason you tossed peer here? Also nit most of the code we work in goes 

private readonly
private 
public props
ctor

for order. Might wanna just do that here for consistency